frontend/src/i18n/locales/en-US.json

```diff
@@ -321,6 +321,7 @@
     "use-public-key-checked": "Sent using recipient's standard public key"
```

Comment:
Working great! But could we either level and center the `Clear history` button with `Send history is only stored locally on this device. [Learn more](http://localhost:8080/faq#why-cant-I-see-my-send-history-on-different-devices)` Or maybe even better to move the button to bottom of the history table.

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/546/commits/abadb4beefef4f798c0547610eb21339362d9840